internal/core/api.rs

```diff
@@ -431,7 +470,8 @@ impl Window {
     }
 }
 
-pub use crate::input::PointerEventButton;
+pub use crate::input::{key_codes, KeyboardModifiers, PointerEventButton};
```

Comment:
i think we need to polish a bit the key_codes API.

Reply:
@ogoffart do you have a specific solution in mind, maybe a `Key` struct?